javascript/ql/src/semmle/javascript/frameworks/Express.qll

```diff
@@ -832,10 +832,6 @@ module Express {
         asExpr().(MethodCallExpr).calls(any(ResponseExpr res), name))
     }
 
-    override DataFlow::Node getDataNode() {
-      result = DataFlow::valueNode(astNode)
```

Comment:
It seems like this class should be a `FileSystemReadAccess` without a data node?

Reply:
This is not a request for a change in this PR; just wondering about how to classify the `sendFile` operation.